src/include/distributed/statistics_collection.h

```diff
@@ -0,0 +1,26 @@
+/*-------------------------------------------------------------------------
+ *
+ * statistics_collection.h
+ *
+ * Copyright (c) 2017, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#ifndef STATISTICS_COLLECTION_H
+#define STATISTICS_COLLECTION_H
+
+/* Config variables managed via guc.c */
+extern bool EnableStatisticsCollection;
+
+#if HAVE_LIBCURL
+
+#define STATS_COLLECTION_HOST "https://citus-statistics.herokuapp.com"
```

Comment:
can we get a citusdata.com domain name for this?

Reply:
Sure, I'll do this in later commits.